concepts/05 UI Components/DataGrid/30 Filtering and Searching/6 API/5 Filtering Columns Bound to Collection/20 Server-Side Filtering.md

```diff
@@ -0,0 +1,163 @@
+In the code below, you can see how to bind DataGrid to the [OData service](/Documentation/Guide/Data_Binding/Data_Source_Examples/#OData) and enable remote filtering. To make this solution work, your OData service should support the **Any** filtering operator (see [Support for Any and All](https://www.odata.org/blog/support-for-any-and-all/)). Also, this concept described at the [Azure: OData Collection Operators](https://learn.microsoft.com/en-us/azure/search/search-query-odata-collection-operators) topic. 
```

Comment:
In the code below -- it's a clause, which adds a break, a comma at the beginning of the sentence. Do we really need it here? There's almost always a way to rephrase the beginning without a clause. And, as with the previous occurrence, it's a quite easy task to accomplish.

Second. You use articles with "DataGrid" inconsistently. You should either use a definite article with DataGrid everywhere or nowhere. The choice is yours, for both options are valid.

Third. A solution can be described in a topic, not at a topic. Moreover, if a reference encompasses more than one word -- we should move it to the very end of a sentence to simplify the grammatical structure, and, therefore, the overall text perception.

P.S. The bracketed "see" looks acceptable though.

```suggestion
The following code example demonstrates how to bind the DataGrid to the [OData service](/Documentation/Guide/Data_Binding/Data_Source_Examples/#OData) and enable remote filtering. To make this solution work, make sure that your OData service supports the **Any** filtering operator (see [Support for Any and All](https://www.odata.org/blog/support-for-any-and-all/)). This concept is also described in the following topic: [Azure: OData Collection Operators](https://learn.microsoft.com/en-us/azure/search/search-query-odata-collection-operators).
```